plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -39,6 +40,12 @@ class ServiceDestroyModal extends React.Component {
     });
   }
 
+  isGroupWithServices() {
```

Comment:
Please move this helper to a dedicated util or add a meaning full getter to the `Tree` struct.

Reply:
Created a new util named ServiceTreeUtil.js. First I added this method in the ServiceUtil.js but it was creating a loop because ServiceTree.js included the ServiceUtil.js and if the ServiceUtil would include ServiceTree.js it would reference ServiceTree to {} to break the recursion.